watertap/unit_models/electrodialysis_1D.py

```diff
@@ -572,6 +633,20 @@ def build(self):
             units=pyunits.dimensionless,
             doc="water recovery ratio calculated by mass",
         )
+        self.velocity_D = Var(
+            self.flowsheet().time,
+            self.diluate.length_domain,
+            initialize=0.01,
+            units=pyunits.meter * pyunits.second**-1,
+            doc="Linear velocity of flow",
+        )
+        self.velocity_C = Var(
+            self.flowsheet().time,
+            self.diluate.length_domain,
+            initialize=0.01,
+            units=pyunits.meter * pyunits.second**-1,
+            doc="Linear velocity of flow",
+        )
```

Comment:
rename to be more specific `velocity_diluate` and `velocity_concentrate`

Reply:
Done.